log-viewer/src/main/java/com/logviewer/data2/Log.java

```diff
@@ -141,6 +148,25 @@ public Snapshot createSnapshot() {
         }
     }
 
+    private void decompressAndCopyGZipFile() throws IOException {
+
+        File tempFile = File.createTempFile("log-viewer-", file.getName(file.getNameCount() - 1) + ".tmp");
```

Comment:
If a user refresh the page, log-viewer will unpack the file again, you have to check if the unpacked file already exists

Reply:
I was wrong, `com.logviewer.data2.Log` object is cached, so the file will not be unpacked twice after refresh of the page.

Reply:
Yes it is ... I checked it already and the file is being reused